nautobot/extras/migrations/0117_task_queue_to_job_queue_migration.py

```diff
@@ -11,10 +12,17 @@ def migrate_task_queues_to_job_queues(apps, schema):
     ScheduledJob = apps.get_model("extras", "ScheduledJob")
     JobQueue = apps.get_model("extras", "JobQueue")
 
+    default_job_queue, _ = JobQueue.object.get_or_create(
+        name=settings.CELERY_TASK_DEFAULT_QUEUE, defaults={"queue_type": JobQueueTypeChoices.TYPE_CELERY}
+    )
+
     for job in Job.objects.all():
         task_queues = job.task_queues
         # remove duplicates?
         task_queues = list(set(task_queues))
+        # Add default Job Queue to every job record.
+        job.job_queues.add(default_job_queue)
```

Comment:
I *think* we should only do this if `task_queues` is an empty list? Not 100% sure.